src/components/Page-1/Page1.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+// import './Page1.module.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
+// import { NavLink } from 'react-router-dom';
```

Comment:
- [ ] Kindly remove the commented-out code to keep the code base clean.

- [ ] Kindly don't disable the linter checks. Linter checks are in place in order to better structure your code and improve your code quality.